src/main/java/de/tum/in/www1/exerciseapp/web/rest/MultipleChoiceQuestionStatisticResource.java

```diff
@@ -0,0 +1,148 @@
+package de.tum.in.www1.exerciseapp.web.rest;
+
+import com.codahale.metrics.annotation.Timed;
+import de.tum.in.www1.exerciseapp.domain.MultipleChoiceQuestionStatistic;
+
+import de.tum.in.www1.exerciseapp.domain.QuizPointStatistic;
+import de.tum.in.www1.exerciseapp.repository.MultipleChoiceQuestionStatisticRepository;
+import de.tum.in.www1.exerciseapp.web.rest.util.HeaderUtil;
+import io.github.jhipster.web.util.ResponseUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * REST controller for managing MultipleChoiceQuestionStatistic.
+ */
+@RestController
+@RequestMapping("/api")
+public class MultipleChoiceQuestionStatisticResource {
+
```

Comment:
Please make sure that all REST calls (also the generated ones) include the common PreAuthorize annotation (this comment is valid for all REST resources changed in this pull request)